README.rst

```diff
@@ -0,0 +1,80 @@
+pizzapi
+=======
+
+This is a Python wrapper for the Dominos Pizza API.
+
+It's a port of `the pizzapi node.js module <https://github.com/RIAEvangelist/node-dominos-pizza-api>`_ written by `RIAEvangelist <https://github.com/RIAEvangelist>`_.
+
+Quick Start
+-----------
+
+First construct a ``Customer`` object and set the customer's address:
+
+.. code-block:: python
+
+    customer = Customer('Barack', 'Obama', 'barack@whitehouse.gov', '2024561111', '700 Pennsylvania Avenue NW, Washington, DC, 20408')
+    address = Address('700 Pennsylvania Avenue NW', 'Washington', 'DC', '20408')
+    # or Address(*customer.address.split(',')) if you're lazy
```

Comment:
glad you noticed my note about initializing Customer and Address separately.  this is exactly what i had in mind, though do we still need to supply the address to Customer object?  doesnt seem required other than having the convenience of calling 'customer.address' just to verify it.

Reply:
Nah, there's got to be a simpler way to do this. However, that wasn't the point of this particular pull request. I'll do that later.

In the future we'll just keep "Address" as an internal object that the user won't have to instantiate directly. I also plan to add [this module](https://github.com/datamade/usaddress) so things will be a lot less strict from the user's perspective.